docs/adding-a-page.md

```diff
@@ -0,0 +1,325 @@
+# addons-frontend
+
+This will outline what is required to add a page to the project. A basic knowledge of
+[react](https://facebook.github.io/react/docs/getting-started.html) and
+[redux](http://redux.js.org/) is assumed.
+
+## Structure
+
+A basic app structure will look like this:
+
+```
+src/
+  <app-name>/
+    components/
+      MyComponent/
+        index.js
+        styles.scss
+    containers/
+      MyContainer/
+        index.js
+        styles.scss
```

Comment:
CSS is currently in its own directory as opposed to being like this next to the JS.